hbase-client/src/main/java/org/apache/hadoop/hbase/client/MultiServerCallable.java

```diff
@@ -76,7 +75,7 @@ protected HRegionLocation getLocation() {
   }
 
   @Override
-  public HRegionInfo getHRegionInfo() {
+  public RegionInfo getRegionInfo() {
```

Comment:
You changing a public method here? Do you have to deprecate the old?

Reply:
Or is this internal, private?

Reply:
This one is marked as private. Per description in the book we can change it without further notice. Because of that, I thought we change it directly instead of deprecating it.